projects/1.20/assets/productivebees/productivebees/lang/zh_cn.json

```
@@ -0,0 +1,1015 @@
+  "entity.productivebees.meitnerium_bee": "䥑蜜蜂",
+  "entity.productivebees.mendelevium_bee": "钔蜜蜂",
+  "entity.productivebees.mercury_bee": "汞蜜蜂",
+  "entity.productivebees.molybdenum_bee": "钼蜜蜂",
+  "entity.productivebees.moscovium_bee": "镆蜜蜂",
+  "entity.productivebees.neodymium_bee": "钕蜜蜂",
+  "entity.productivebees.neon_bee": "氖蜜蜂",
+  "entity.productivebees.neptunium_bee": "镎蜜蜂",
+  "entity.productivebees.nihonium_bee": "镍蜜蜂",
+  "entity.productivebees.nitrogen_bee": "氮蜜蜂",
+  "entity.productivebees.nobelium_bee": "锘蜜蜂",
+  "entity.productivebees.niobium_bee": "铌蜜蜂",
+  "entity.productivebees.oganesson_bee": "鿫蜜蜂",
+  "entity.productivebees.oxygen_bee": "氧蜜蜂",
+  "entity.productivebees.palladium_bee": "钯蜜蜂",
+  "entity.productivebees.phosphorus_bee": "磷蜜蜂",
+  "entity.productivebees.plutonium_bee": "钚蜜蜂",
+  "entity.productivebees.polonium_bee": "钋蜜蜂",
+  "entity.productivebees.potassium_bee": "钾蜜蜂",
+  "entity.productivebees.praseodymium_bee": "镨蜜蜂",
+  "entity.productivebees.promethium_bee": "钷蜜蜂",
+  "entity.productivebees.protactinium_bee": "镤蜜蜂",
+  "entity.productivebees.radium_bee": "镭蜜蜂",
+  "entity.productivebees.radon_bee": "氡蜜蜂",
+  "entity.productivebees.rhenium_bee": "铼蜜蜂",
+  "entity.productivebees.rhodium_bee": "铑蜜蜂",
+  "entity.productivebees.roentgenium_bee": "𬬭蜜蜂",
+  "entity.productivebees.rubidium_bee": "铷蜜蜂",
+  "entity.productivebees.ruthenium_bee": "钌蜜蜂",
+  "entity.productivebees.rutherfordium_bee": "鑪蜜蜂",
+  "entity.productivebees.samarium_bee": "钐蜜蜂",
+  "entity.productivebees.scandium_bee": "钪蜜蜂",
+  "entity.productivebees.seaborgium_bee": "𨭎蜜蜂",
+  "entity.productivebees.selenium_bee": "硒蜜蜂",
+  "entity.productivebees.silicium_bee": "硅蜜蜂",
+  "entity.productivebees.sodium_bee": "钠蜜蜂",
+  "entity.productivebees.strontium_bee": "锶蜜蜂",
+  "entity.productivebees.tantalum_bee": "钽蜜蜂",
+  "entity.productivebees.technetium_bee": "锝蜜蜂",
+  "entity.productivebees.tellurium_bee": "碲蜜蜂",
+  "entity.productivebees.tennessine_bee": "鿬蜜蜂",
+  "entity.productivebees.terbium_bee": "铽蜜蜂",
+  "entity.productivebees.thallium_bee": "铊蜜蜂",
+  "entity.productivebees.thorium_bee": "钍蜜蜂",
+  "entity.productivebees.thulium_bee": "铥蜜蜂",
+  "entity.productivebees.vanadium_bee": "钒蜜蜂",
+  "entity.productivebees.xenon_bee": "氙蜜蜂",
+  "entity.productivebees.ytterbium_bee": "镱蜜蜂",
+  "entity.productivebees.yttrium_bee": "钇蜜蜂",
+  "entity.productivebees.zirconium_bee": "锆蜜蜂",
+  "entity.productivebees.water_bee": "水蜜蜂",
+  "entity.productivebees.gregshell_bee": "全金属帕特里克蜜蜂",
+  "entity.productivebees.cyanite_bee": "蓝晶蜜蜂",
+  "entity.productivebees.blutonium_bee": "蓝钚蜜蜂",
+  "entity.productivebees.magentite_bee": "磁铁矿蜜蜂",
+  "entity.productivebees.ludicrite_bee": "纯镥蜜蜂",
+  "entity.productivebees.ridiculite_bee": "重晶石蜜蜂",
+  "entity.productivebees.inanite_bee": "无生命蜜蜂",
+  "entity.productivebees.insanite_bee": "含水蜜蜂",
+  "entity.productivebees.arcane_crystal_bee": "奥秘蜜蜂",
+  "entity.productivebees.deorum_bee": "迪奥鲁姆蜜蜂",

```

```suggestion
  "entity.productivebees.deorum_bee": "圣金蜜蜂",
```
Forbidden & Arcanus的圣金